fxa-auth-db-server/index.js

```diff
@@ -192,6 +146,52 @@ function createServer(db) {
     }
   )
 
+  function handleSuccess(req, res, result) {
```

Comment:
@vbudhram had to move this because eslint detected that `api` was used before assignment